### ISSUE TITLE
Skirmish AI is no longer preparing resources for missions.

### ISSUE THREAD

**Issue by stefanhendriks** [stefanhendriks/Dune-II---The-Maker#226]
I noticed that the AI produces missions, but does not seem to pass the required things to build in the buildOrders queue. Hence, the AI does nothing?

It must have broken very recently.

**Comment by stefanhendriks** [comment]
looks like the AI would throw away build orders which it can't build because it didn't have the required structure. 

Now it won't do that.

Additionaly, to prevent waiting forever for resource gathering, I have set a time limit. So when that passes, the mission will continue to be executed.

Also, when the AI has all missions in "execute" mode, it will build additional (random) units, so it won't start building them once a mission is over and restarted. But only after 5 minutes in-game, and when it has enough money to do so.

**Comment by stefanhendriks** [comment]
After more testing it looks like the AI is still too passive. 

Also, I noticed the super weapons no longer get fired by AI? (an enemy had missile ready, but didnt fire it).
